master/reference/node/configuration.md

```diff
@@ -176,19 +177,18 @@ IP6_AUTODETECTION_METHOD=interface=eth.*
 
 The `calico/node` container supports an exec readiness endpoint.
 
-Here is the help text:
+To access this endpoint, use the following command.
 
 ```
-$ docker exec calico-node /bin/readiness -h
-Usage of dist/readiness:
-  -bird
-    	Specify to enable BIRD readiness checks
-  -bird6
-    	Specify to enable BIRD6 readiness checks
-  -felix
-    	Specify to enable felix readiness checks
+docker exec calico-node /bin/calico-node [flag]
 ```
 
-The BIRD readiness endpoint ensures that the BGP mesh is healthy by verifiying that all BGP peers are established and
+Substitute `[flag]` with any of the following.
```

Comment:
it's really any combination of the following.

e.g. `docker exec calico-node /bin/calico-node -bird-ready -felix-ready` will check both.